### PR TITLE
variable 'controller' collides with imported package name

### PR DESCRIPTION
**Pull request by StudyNick**
variable 'controller' collides with imported package name.

improvement:
'controller' modify 'rc'

### CODE REVIEW

**Comment by StudyNick** [comment]
I signed it!

**Comment by caesarxuchao** [comment]
@k8s-bot ok to test

**Comment by caesarxuchao** [comment]
Thanks @StudyNick. LGTM.

File: pkg/controller/replication/replication_controller.go [hunk 1]
```diff
@@ -179,14 +179,14 @@ func (rm *ReplicationManager) Run(workers int, stopCh <-chan struct{}) {
 func (rm *ReplicationManager) getPodController(pod *v1.Pod) *v1.ReplicationController {
 	// look up in the cache, if cached and the cache is valid, just return cached value
 	if obj, cached := rm.lookupCache.GetMatchingObject(pod); cached {
-		controller, ok := obj.(*v1.ReplicationController)
```
Comment (mikedanese) [inline pkg/controller/replication/replication_controller.go]: just rc, not podController

Comment (StudyNick) [inline pkg/controller/replication/replication_controller.go]: yes ,i agree. I have already modified

Comment (StudyNick) [inline pkg/controller/replication/replication_controller.go]: @mikedanese  yes ,i agree. I have already modified

Comment (StudyNick) [inline pkg/controller/replication/replication_controller.go]: modify "rc"

**Comment by mikedanese** [comment]
Please squash.

**Comment by StudyNick** [comment]
@mikedanese it is ok.

**Comment by mikedanese** [comment]
You haven't squashed yet.

**Pull request closed without merging**